care/users/api/serializers/user.py

```diff
@@ -52,6 +57,39 @@ def create(self, validated_data):
         validated_data["password"] = make_password(validated_data.get("password"))
         return super().create(validated_data)
 
+    def validate(self, attrs):
+        validated = super().validate(attrs)
+        if attrs["user_type"] == "Doctor":
+            if not attrs["doctor_qualification"]:
```

Comment:
are you sure that the field will be available in `attrs`

Reply:
Yes. Am I missing context on anything?
I've tested the front-end locally by creating a non-doctor user and a user of type doctor.

Reply:
what if the value `doctor_qualification` is not sent in the API, remember that validations must be present in both the BE and FE.

Reply:
@vigneshhari  
- [x] Validations for front-end already added
- [ ] for the backend, I'll change it to use `attrs.get(...)` instead

Reply:
@vigneshhari done. Also, I've tested with the front-end of develop branch locally which does not include the new fields.
Works :)

![image](https://user-images.githubusercontent.com/25143503/213846463-df200b4b-a751-4cd1-a261-f18eb38a232d.png)

Reply:
![image](https://user-images.githubusercontent.com/25143503/213846532-668914fa-8785-4df5-90cd-1c9b147dce58.png)